pkg/agent/watch.go

```diff
@@ -376,6 +397,10 @@ func updateVMMetrics(perVMMetrics *PerVMMetrics, oldVM, newVM *vmv1.VirtualMachi
 		updateMetrics(spec.gauge, oldMetrics, newMetrics)
 	}
 
+	oldExtraIPMetrics := makeVMExtraIPMetrics(oldVM)
+	newExtraIPMetrics := makeVMExtraIPMetrics(newVM)
+	updateMetrics(perVMMetrics.extraIP, oldExtraIPMetrics, newExtraIPMetrics)
```

Comment:
question(blocking): Aren't these already included in `getGaugeSpecs` ?

Reply:
Oops, sorry about that, I changed the order of the commits, so the old way and new way to add metrics got mixed up.